build.gradle

```diff
@@ -44,8 +44,9 @@ dependencies {
     compile "io.dropwizard:dropwizard-auth:${dropwizardVersion}"
     compile "io.dropwizard:dropwizard-client:${dropwizardVersion}"
     compile "de.thomaskrille:dropwizard-template-config:1.4.0"
-    // compile "io.dropwizard:dropwizard-jdbi:${dropwizardVersion}"
-    // compile ':ojdbc6:'
+    compile "io.dropwizard:dropwizard-jdbi:${dropwizardVersion}"
+    compile ':ojdbc6:'
```

Comment:
Could you update this to ojdbc8? We recently updated our apis to this because it supports more hashing algorithms for encrypted connections to databases. ojdbc6 only supports sha1. It should just be a one line change here, and then the new jar file https://github.sig.oregonstate.edu/ecs-data/ansible-private-roles/blob/master/roles/api_db_files/files/bin/ojdbc8.jar